packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -418,7 +418,13 @@
             "lazy": {
               "type": "boolean",
               "description": "If the bundle will be lazy loaded.",
-              "default": false
+              "default": false,
+              "x-deprecated": "Use 'inject' instead."
```

Comment:
Can you change this to `Use the 'inject' option with 'false' value instead.` please? Just changing `lazy` to `inject` won't be what users want.